tests/conftest.py

```diff
@@ -1,11 +1,29 @@
 import pytest
 
+import gitlab
+
 
 @pytest.fixture(scope="session")
 def test_dir(pytestconfig):
     return pytestconfig.rootdir / "tests"
 
 
+@pytest.fixture(autouse=True)
+def mock_clean_config(monkeypatch):
+    """Ensures user-defined environment variables do not interfere with tests."""
+    monkeypatch.delenv("PYTHON_GITLAB_CFG", raising=False)
+    monkeypatch.delenv("GITLAB_PRIVATE_TOKEN", raising=False)
+    monkeypatch.delenv("GITLAB_URL", raising=False)
+    monkeypatch.delenv("CI_JOB_TOKEN", raising=False)
+    monkeypatch.delenv("CI_SERVER_URL", raising=False)
```

Comment:
Not really needed because of tox, but thought I'd add it anyway since we were doing it in other parts.